ecleankernel/file.py

```diff
@@ -96,6 +97,25 @@ def __init__(self,
         super().__init__(path, KernelFileType.KERNEL)
         self.internal_version = self.read_internal_version()
 
+    def decompress_lzma(self, data):
+        results = []
+        while True:
+            decomp = LZMADecompressor(FORMAT_AUTO, None, None)
+            try:
+                res = decomp.decompress(data)
+            except LZMAError:
+                if results:
+                    break  # Leftover data is not a valid LZMA/XZ stream; ignore it.
```

Comment:
What's the value of `decomp.unused_data` at this point?

Reply:
It is an empty byte.
![image](https://user-images.githubusercontent.com/38039130/158626402-12529402-961d-4a69-8b42-c707093f58c0.png)